doc/api/util.markdown

```diff
@@ -53,8 +53,7 @@ argument. Supported placeholders are:
 
 * `%s` - String.
 * `%d` - Number (both integer and float).
-* `%j` - JSON.  Replaced with the string `'[Circular]'` if the argument
-         contains circular references.
+* `%j` - JSON.  Replaced with the string `'[Circular]'` if the argument contains circular references.
```

Comment:
This line is over 80 characters now. Could you just remove the indentation from the original text?